Python-path-traversal-files-main/DannyWuzhiheng/Web/IP.py

```
@@ -0,0 +1,303 @@
+        rpassword = request.form['rpassword'] 
+        sig=request.form['signature'] 
+        with open(r'./login.txt','r') as f:
+            f.read()
+            for i in a:
+                usps=i.split(' ')
+                if usps[0]==username:
+                    return "注册失败，用户名已存在" 
+        with open(r'./login.txt','a') as f:
+            if password==rpassword:
+                f.write(f'\n{username} {password} {sig}')
+                os.system(f"cp pic/user/main.png pic/user/upload/{username}.png")
```

Ruff (0.13.3)

Starting a process with a shell, possible injection detected

(S605)